### DISCUSSION TITLE
Donations?

### DISCUSSION THREAD

**Discussion by zouharvi** [typst/typst#252] [General]
(originally an [issue](https://github.com/typst/typst/issues/214))

Thanks for the amazing project! My sentiment about it is [pretty clear](https://twitter.com/zouharvi/status/1638870887028989953). I wanted to inquire whether there is a way to sponsor this project financially.

I understand that it may be tricky to set it up, since this project is sponsored by EU/city of Berlin.

**Comment by Enivex** [comment]
Can we wish for the success of this project without wishing for the death of LaTeX?

**Reply by zouharvi** [reply]
It's just a blurb-Tweet that resonates with some people in the academia. :slightly_smiling_face: 

I don't want to get off-topic here but it's clear that LaTeX and Typst would occupy the same niche in academia, which makes it zero-sum.

**Reply by Enivex** [reply]
> It's just a blurb-Tweet that resonates with some people in the academia. 🙂
> 
> I don't want to get off-topic here but it's clear that LaTeX and Typst would occupy the same niche in academia, which makes it zero-sum.

I don't see why they cannot coexist. Competition is always good.

At least in mathematics, I don't see Typst displacing Latex for _many_ years. The amount of labor that has been put into some packages is truly overwhelming. The PGF/TikZ manual alone is 1321 pages (!), and the closely related PGFPlots is at 573 pages. That's almost 1900 pages for graphics packages, not even including all the other PGF/TikZ libraries.

In particular, changing the syntax for math input will be a really tough sell (possibly even a deal-breaker) when it's been ingrained over decades. A possible route towards easing adoption would be to offer a an alternate LaTeX-style math input.

**Reply by zouharvi** [reply]
Perl also had a huge ecosystem of packages, dependencies, literature, manuals, users 20 years ago.

It's just my personal opinion, I don't mean to start a flame war (and also to distract this thread from the original question).

**Reply by jorsn** [reply]
A problem with latex is also that you *need* so many packages, because almost nothing is trivial.
I think, the main obstacles are conservative journals accepting LaTeX and maybe Word (who don't want to or can change their processes, e.g. ScholarOne publishing software uses TeXlive 2013, still), and conservative academics (who, understandably, want to focus on content, not tools).

**Comment by zouharvi** [comment]
Seems to be resolved via [commit 0d1037a](https://github.com/typst/typst/commit/0d1037a58c534c1ef26e69d430f872b82f465378), which adds the sponsorship button to the GitHub page.
